ci/rebuild_updated_recipes.py

```diff
@@ -74,21 +77,24 @@ def build(target_python, requirements):
 def main():
     target_python = TargetPython.python3
     recipes = modified_recipes()
-    print('recipes modified:', recipes)
+    logger.info('recipes modified: {}'.format(recipes))
     recipes -= CORE_RECIPES
-    print('recipes to build:', recipes)
+    logger.info('recipes to build: {}'.format(recipes))
     context = Context()
-    build_order, python_modules, bs = get_recipe_order_and_bootstrap(
-        context, recipes, None)
-    # fallback to python2 if default target is not compatible
-    if target_python.name not in build_order:
-        print('incompatible with {}'.format(target_python.name))
+    # forces the default target
+    recipes_and_target = recipes | set([target_python.name])
```

Comment:
required because of https://github.com/kivy/python-for-android/blob/4ed9dff/pythonforandroid/graph.py#L131